docs/main/user-guides/uninstallation.md

```diff
@@ -30,6 +30,10 @@ Run the following command to uninstall OpenEBS:
 helm uninstall openebs -n <OPENEBS_NAMESPACE>
 ```
 
+:::note
+Uninstalling the Helm chart does not remove/uninstall the CustomResourceDefinitions (CRDs).
```

Comment:
Should we specify the CRDs? CC @niladrih

Reply:
We could have a code block just below it
```
diskpools.openebs.io
volumesnapshotclasses.snapshot.storage.k8s.io
volumesnapshotcontents.snapshot.storage.k8s.io
volumesnapshots.snapshot.storage.k8s.io
```